src/helm/benchmark/presentation/summarize.py

```diff
@@ -998,21 +1010,30 @@ def _adapter_spec_sort_key(spec):
             runs = adapter_to_runs[adapter_spec]
             display_name = get_method_display_name(model_metadata.display_name, info)
 
-            # Link to all the runs under this model
-            if link_to_runs:
+            # Link the runs that this row was aggregated from, if this is a scenario table.
+            # Scenario tables link to the runs in the model cells,
+            # whereas non-scenario tables link to the runs in the metrics cells.
+            run_spec_names: Optional[List[str]]
+            if is_scenario_table:
                 run_spec_names = [run.run_spec.name for run in runs]
                 href = run_spec_names_to_url(run_spec_names)
             else:
+                run_spec_names = None
                 href = None
 
             # Render contamination information
             point = self.contamination.get_point(model_name, columns[0][0].name)
             if num_groups == 1 and point is not None:  # display contamination information at the adapter level
                 cells = [
-                    Cell(display_name + CONTAMINATION_SYMBOLS[point.level], description=point.description, href=href)
+                    Cell(
+                        display_name + CONTAMINATION_SYMBOLS[point.level],
+                        description=point.description,
+                        href=href,
+                        run_names=run_spec_names,
+                    )
                 ]
             else:
-                cells = [Cell(display_name, description="", href=href)]
+                cells = [Cell(display_name, description="", href=href, run_names=run_spec_names)]
```

Comment:
Can we call it `run_spec_names` instead of `run_names` for consistency everywhere?

Reply:
Sure, changed `run_names` to `run_spec_names`.